### DISCUSSION TITLE
emulator 64 bit

### DISCUSSION THREAD

**Discussion by jeuxdaily** [airsdk/Adobe-Runtime-Support#2309] [General]
Hi
any emulator how can work apk publising with sdk air 33.1.1.620 and latest
I try some emulator without any result

**Comment by ajwfrost** [comment]
Hi

We use emulators a lot, without any issues. Generally best to set one up using Android Studio / AVD and then going for the optimal processor type (x86_64 normally). You then need to ensure the AIR app matches this architecture so would need to add the `-arch x64` argument when creating the APK.

Other than that, the detection, deployment and running of apps on an Android emulator should all work fine, which stage are you seeing a failure with?

thanks

**Comment by jeuxdaily** [comment]
thanks for your help
I disabled this line on adt.cfg to test apk in emulator

# To override the creation of an Android APK file, and to instead create an Android App Bundle, the below
# flag can be uncommented. Note that the bundle may end up with an "apk" file extension
#CreateAndroidAppBundle=true

it's working fine
